spec/support/shared_validator_specs.rb

```diff
@@ -16,6 +16,31 @@ def alphanumeric_string(length)
   end
 end
 
+shared_examples "a property with data-types restriction" do
```

Comment:
"a property with validation" would be strange because it's all about validation, too generic. How about "a property with a value validation/restriction"?